NeoMathEngine/src/GPU/Vulkan/VulkanMathEngine.cpp

```diff
@@ -83,8 +81,9 @@ inline int Ceil( int val, int discret )
 
 const int VulkanMemoryAlignment = 16;
 
-CVulkanMathEngine::CVulkanMathEngine( CVulkanDll& _dll, int _deviceNumber, size_t memoryLimit ) :
-	dll( _dll ),
+CVulkanMathEngine::CVulkanMathEngine( int _deviceNumber, size_t memoryLimit ) :
+	dllLoader( CDllLoader::VULKAN_DLL ),
+	dll( *CDllLoader::vulkanDll ),
 	deviceNumber( _deviceNumber ),
 	device( dll.CreateDevice( dll.GetDevices()[_deviceNumber] ) )
```

Comment:
Probably, it's better to create device before this constructor (in order to avoid ASSERT_EXPR when something goes wrong and return nullptr from `GpuManager::CreateMathEngine`)

Reply:
+1 fix it please